includes/CPT/class-issue.php

```diff
@@ -0,0 +1,252 @@
+<?php
+/**
+ * Govpack
+ *
+ * @package Newspack
+ */
+
+namespace Newspack\Govpack\CPT;
+
+use \Newspack\Govpack\Helpers;
+
+/**
+ * Register and handle the "Issue" Custom Post Type
+ */
+class Issue {
+
+	/**
+	 * Valid issue formats.
+	 *
+	 * @var array
+	 */
+	public static $issue_formats = [ 'full', 'mini', 'wiki' ];
```

Comment:
I don't think this applies to issues — it's for profiles. @designhawg do we have any designs for issue pages?

Reply:
Formats have now been removed from "Issues".